vmf/scripts/mods/vmf/modules/ui/options/vmf_options_view.lua

```diff
@@ -526,39 +526,39 @@ local menu_widgets_definition = {
     scenegraph_id = "sg_root",
     element = {
       passes = {
-          {
-            pass_type = "rect",
-
-            style_id  = "background_border"
-          },
-          {
-            pass_type = "rect",
-
-            style_id  = "background_settings_list"
-          },
-          {
-            pass_type = "texture",
+        {
+          pass_type = "rect",
+          style_id = "background",
+          texture_id = "rect_masked_texture"
+        },
+        {
+          pass_type = "texture_frame",
+          style_id = "frame",
+          texture_id = "frame"
```

Comment:
```suggestion
          texture_id = "frame",
```

Opinionated change: dangling commas make code easier to handle.
Rationale: https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8

Reply:
for sure, I do that in JavaScript was not sure it was supported in Lua. In python it can change types O.o